src/Admin/FormCaptureAdmin.php

```diff
@@ -37,26 +37,32 @@ class FormCaptureAdmin extends ModelAdmin
 
     private static $menu_icon_class = 'font-icon-checklist';
 
-    public function getEditForm($id = null, $fields = null)
-	{
-		$form = parent::getEditForm();
+    public function getGridField(): GridField
+    {
+        $field = parent::getGridField();
+        $field->setName('Submissions');
 
-		/** @var GridField $gridField */
-		$gridField = $form->Fields()->dataFieldByName($this->sanitiseClassName($this->modelClass));
-		$gridField->setName('Submissions');
+        $this->extend('updateGridField', $field);
 
-		$config = $gridField->getConfig();
-		$config->removeComponentsByType(GridFieldImportButton::class);
-		$config->removeComponentsByType(GridFieldPrintButton::class);
+        return $field;
+    }
 
-		// Configure filtering options
+    public function getGridFieldConfig(): GridFieldConfig
+    {
+        $config = parent::getGridFieldConfig();
```

Comment:
Same as above - I think this will call the `updateGridFieldConfig` hook so it’ll end up being called twice when it’s called at the end of this function.